test/parallel/test-repl-persistent-history.js

```diff
@@ -74,19 +75,19 @@ const sameHistoryFilePaths = '\nThe old repl history file has the same name ' +
                              path.join(common.tmpDir, '.node_repl_history') +
                              ' and is empty.\nUsing it as is.\n';
 // File paths
-const fixtures = common.fixturesDir;
-const historyFixturePath = path.join(fixtures, '.node_repl_history');
+const fixturesDir = fixtures.fixturesDir;
+const historyFixturePath = path.join(fixturesDir, '.node_repl_history');
```

Comment:
This is good but can be further simplified using the `fixtures.path()` function, e.g. 

```js
const historyFixturePath = fixtures.path('.node_repl_history');
```

Reply:
done!